Gemfile

```diff
@@ -24,6 +24,8 @@ gem 'twilio-ruby',             '~> 4.11.1'
 #gem "taps"
 #gem 'validates_timeliness', '~> 3.0'
 gem 'devise'
+gem 'puma'
```

Comment:
What is the purpose of puma here ?

Reply:
Webrick doesn't support of action cable, but puma does.  If we'd like live updating of the indicator in the future, there will need to be a switch.  Until then we won't need it though, I will take it out.